Source/WebCore/dom/ViewTransition.h

```diff
@@ -159,10 +160,13 @@ class ViewTransition : public RefCounted<ViewTransition>, public CanMakeWeakPtr<
 
     bool documentElementIsCaptured() const;
 
+    const ViewTransitionTypeSet& types() const { return m_types; }
+    void setTypes(Ref<ViewTransitionTypeSet>&& newTypes) { m_types = newTypes; }
```

Comment:
```suggestion
    void setTypes(Ref<ViewTransitionTypeSet>&& newTypes) { m_types = WTFMove(newTypes); }
```

Reply:
I forgot that you need to explicitly use `WTFMove` in this scenario 😅